spec/car_spec.rb

```diff
@@ -3,38 +3,44 @@
 
 RSpec.describe Car do
   it 'exists' do
-    car = Car.new("Ford Mustang", 1500, 36)
```

Comment:
Unfortunately this was the right approach - there should only be 3 arguments in your `car#initialize` method. Separating your first arg into 2 goes against the interaction pattern where all the cars have both make & model in one string. The expectation is to then separate them by the space via `.split`.